src/main/kotlin/org/opensearch/commons/alerting/action/GetFindingsRequest.kt

```diff
@@ -6,26 +6,42 @@ import org.opensearch.commons.alerting.model.Table
 import org.opensearch.core.common.io.stream.StreamInput
 import org.opensearch.core.common.io.stream.StreamOutput
 import java.io.IOException
+import java.time.Instant
 
 class GetFindingsRequest : ActionRequest {
     val findingId: String?
     val table: Table
     val monitorId: String?
     val monitorIds: List<String>?
     val findingIndex: String?
+    val severity: String?
+    val detectionType: String?
```

Comment:
Same comment on if these should be enums

Reply:
yes, that's right `detectorType` and `logType` are synonyms. We need to make it uniform across the codebase. There should be a seperate PR for that. Yes, severity is stored as `String`

Reply:
Also, detectionType is is `different` from `detectorType/logType.` The detection type can threat-intel and rule based. The UI has a drop down for that and with this enhancement we are doing the filtering at the backend

Reply:
Ah thank you for clearing that up.